src/vivarium_public_health/risks/implementations/low_birth_weight_and_short_gestation.py

```diff
@@ -265,3 +267,178 @@ def _get_current_exposure(self, index: pd.Index) -> pd.DataFrame:
             f" refer directly one of the exposure columns. During simulant initialization the birth"
             f" exposure pipelines should be used instead."
         )
+
+
+class LBWSGRiskEffect(RiskEffect):
+
+    TMREL_BIRTH_WEIGHT_INTERVAL: pd.Interval = pd.Interval(3500.0, 4500.0)
```

Comment:
is this not stored in the data just because it will never change?

Reply:
Good question, also, not being familiar, what is the significance of these particular values? Maybe a comment or constant assignment is in order?

Reply:
This has to do with interpolation. The categorical risk effects for all categories in the area will have RR == 1, but when we perform an interpolation, that would no longer be true so that the surface would be continuous. This introduces a discontinuity at the boundary of TMREL which is not reflected in the interpolator itself. You can see how this is used in `get_relative_risk_for_age_group` called in `on_initialize_simulants`.

Reply:
^ should that detail be noted in the code?